### ISSUE TITLE
Add a plugin that ensures copyright messages are present in all applicable source files

### ISSUE THREAD

**Issue by minborg** [OpenHFT/Chronicle-Bytes#242]
Most source files are missing copyright notices.

Here is how a solution could look like:

```
                    <plugin>
                        <groupId>com.mycila</groupId>
                        <artifactId>license-maven-plugin</artifactId>
                        <version>4.1</version>

                        <configuration>
                            <header>license_header.txt</header>
                            <properties>
                                <currentYear>2021</currentYear>
                            </properties>
                            <headerDefinitions>
                                <headerDefinition>license_header_def.xml</headerDefinition>
                            </headerDefinitions>
                            <excludes>
                                <exclude>**/README</exclude>
                                <exclude>**/README.md</exclude>
                                <exclude>**/package-info.java</exclude>
                                <exclude>**/*.versionsBackup</exclude>
                                <exclude>CONTRIBUTOR_LICENSE_AGREEMENT</exclude>
                                <exclude>LICENSE</exclude>
                                <exclude>DISCLAIMER</exclude>
                                <exclude>THIRD_PARTY_LICENSES</exclude>
                                <exclude>src/test/resources/**</exclude>
                                <exclude>src/main/resources/**</exclude>
                            </excludes>
                        </configuration>

                        <executions>
                            <execution>
                                <goals>
                                    <goal>check</goal>
                                </goals>
                            </execution>
                        </executions>
                    </plugin>
```

**Comment by RobAustin** [comment]
@j4sm1ne96 please ask @minborg for help on how to do this.

**Comment by hft-team-city** [comment]
Released in [Chronicle-Bytes-2.23ea10](https://github.com/OpenHFT/Chronicle-Bytes/releases/chronicle-bytes-2.23ea10), [BOM-2.23ea68](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.23ea68)
